lib/node_modules/@stdlib/utils/some-own-by/docs/repl.txt

```diff
@@ -0,0 +1,45 @@
+
+{{alias}}( obj, n, predicate[, thisArg ] )
+    Tests whether some `own` properties of a provided object
+    satisfies a predicate function for at least `n` properties.
+
+    The predicate function is provided three arguments:
+
+    - `value`: object value
+    - `key`: object key
+    - `obj`: the input object
+
+    The function immediately returns upon finding `n` successful properties.
+
+    If provided an empty object, the function returns `false`.
+
+    Parameters
+    ----------
+    obj: Object
+        Input object over which to iterate.
+
+    n: number
+        Minimum number of successful properties.
+
+    predicate: Function
+        The test function.
+
+    thisArg: any (optional)
+        Execution context.
+
+    Returns
+    -------
+    bool: boolean
+        The function returns `true` if an object's own properties satisfy a 
+        predicate
+        for at least `n` properties; otherwise, the function returns `false`.
```

Comment:
```suggestion
        The function returns `true` if an object's own properties satisfy a 
        predicate for at least `n` properties; otherwise, the function 
        returns `false`.
```